Formula/kafkacat.rb

```diff
@@ -2,8 +2,8 @@ class Kafkacat < Formula
   desc "Generic command-line non-JVM Apache Kafka producer and consumer"
   homepage "https://github.com/edenhill/kafkacat"
   url "https://github.com/edenhill/kafkacat.git",
```

Comment:
GitHub URLs should be updated to https://github.com/edenhill/kcat since the existing URLs redirect there.

May want to consider renaming formula too?

Reply:
yeah, that is a good point